src/sentry/buffer/redis.py

```diff
@@ -208,6 +208,63 @@ def get(
             col: (int(results[i]) if results[i] is not None else 0) for i, col in enumerate(columns)
         }
 
+    def get_redis_connection(self, key: str) -> Any | None:
+        # TODO type RedisPipeline instead of using Any here
```

Comment:
I'm not actually sure how one would go about typing something like this

Reply:
You may be able to import the RedisPipeline type from thee redis package itself, that or maybe we could install a typings package for redis.

Reply:
I'm not seeing it available from importing redis :/

Reply:
idk if this "right" or not with python, but: https://pypi.org/project/types-redis/ - then we could just use that dependency and version it with redis.

That's how I'm used to handling these kinds of types in javascript land, but there's often more FE dependencies than Python deps, so I'm not sure about the appetite to add type here 🤷

Reply:
I'm not sure how to make this call, typically we try be conservative about adding any dependencies. I'll ask somewhere in Slack 🤷‍♀️